Tests/test_file_wal.py

```diff
@@ -0,0 +1,23 @@
+from helper import unittest, PillowTestCase
+
+from PIL import WalImageFile
+
+
+class TestFileWal(PillowTestCase):
+
+    def test_open(self):
+        # Arrange
+        TEST_FILE = "Tests/images/hopper.wal"
+
+        # Act
+        im = WalImageFile.open(TEST_FILE)
+
+        # Assert
+        self.assertEqual(im.format, "WAL")
+        self.assertEqual(im.format_description, "Quake2 Texture")
```

Comment:
Just a question - why assert the `format_description`? Once the `format` is asserted, this is a given. `format_description` isn't tested anywhere else.